timemachine/fe/system.py

```diff
@@ -129,29 +135,22 @@ def get_U_fn(self):
         """
         Return a jax function that evaluates the potential energy of a set of coordinates.
         """
+        assert self.torsion
+        U_fns = self.get_U_fns()
 
         def U_fn(x):
-            assert self.torsion
-
-            # Chiral restraints are disabled until checks are added (see GH #815)
-            # chiral_U = chiral_atom_U(x) + chiral_bond_U(x)
-            return (
-                self.bond(x, box=None)
-                + self.angle(x, box=None)
-                + self.torsion(x, box=None)
-                + self.nonbonded(x, box=None)
-                # + self.chiral_atom(x, box=None)
-                # + self.chiral_bond(x, box=None)
-            )
+            return jnp.sum(jnp.array([U(x, box=None) for U in U_fns]))
```

Comment:
nit: might want to check that this isn't introducing a performance bottleneck. I think the computation on this line is dominated by `jnp.array`, which can be relatively slow (especially if not jitted). In this example, summing the python list directly is ~500x faster:
![image](https://github.com/proteneer/timemachine/assets/319411/56ddba14-47bc-4da0-9f9d-e766121c7002)

Reply:
Reverted in https://github.com/proteneer/timemachine/pull/1243/commits/de851032534c1a0668497e9e3195b70e97f4d118 to be on the safe side